firstvoices/backend/serializers/site_data_serializers.py

```diff
@@ -108,7 +108,8 @@ def get_img(dictionaryentry):
         # NOTE: MTD currently only allows one image. As a heuristic, I'm selecting the first one.
         image = dictionaryentry.related_images.first()
 
-        if image is None:
+        # If no small image is available (possibly due to the wrong file type), return None
+        if image is None or image.small is None:
```

Comment:
minor: should we make it so, that if image.small is None, but the image is present (probably cause of some issue with thumbnail generation), we still send that as a backup ?

Reply:
As per our discussion, if an image does not have a small thumbnail it likely is bad data in the first place, such as an unsupported file type. To avoid setting bad data as images in the app we will only return the small thumbnails.